### DISCUSSION TITLE
signal-cli fails without error message

### DISCUSSION THREAD

**Discussion by whnr** [AsamK/signal-cli#434] [Q&A]
Hi!

* I'm trying to run signal-cli on [uberspace](https://uberspace.de/de/) a CentOs 7 System `Linux faye.uberspace.de 3.10.0-1062.4.1.el7.x86_64 #1 SMP Fri Oct 18 17:15:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`
* The users have limited access rights (no sudo).
* `openjdk version "1.8.0_242"
OpenJDK Runtime Environment (build 1.8.0_242-b08)
OpenJDK 64-Bit Server VM (build 25.242-b08, mixed mode)`

When I try something like this `signal-cli ‪-u "+11111111" register` I only get a response like this:

````
signal-cli:   error:   invalid   choice:   '‪-u'   (choose   from   'trust',   'receive',
'sendContacts', 'updateProfile', 'listIdentities',  'link',  'unregister', 'updateGroup',
'quitGroup',  'listDevices',   'removeDevice',   'updateAccount',   'daemon',   'setPin',
'addDevice', 'listGroups', 'verify', 'updateContact', 'send', 'removePin', 'register')

Did you mean:
        trust
````

Any idea how to add more verbosity, to find the problem?

**Comment by AsamK** [comment]
that's a really strange behavior.
The only way I can trigger a similar behavior on my machine is by using '_u' instead of '-u', so looks like something with the argument prefix detection isn't working correctly.
signal-cli uses argparse4j for its argument parsing.
How did you install signal-cli, could it somehow be using a different argparse4j version?

**Comment by whnr** [comment]
Could it be CentOS? I tried installing via the three step git clone from the repo.
